routes/api/upload.js

```diff
@@ -21,7 +23,16 @@ exports.upload = {
       payload(request, done) {
         done(null, request.payload);
       },
-      filepath(payload, done) {
+      restrict(settings, filename, done) {
```

Comment:
does this happen after the file has been saved?  if so, we need to clean up.  ideally this would happen before the file has been saved.  anything built into hapi?